tests/experimental/post_training/test_low_opset_model.py

```diff
@@ -0,0 +1,128 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+# pylint: disable=redefined-outer-name
+
+import pytest
+
+import os
+
+import torch
+from torchvision import models
+import onnx
+
+from nncf.experimental.onnx.datasets.imagenet_dataset import infer_input_shape
+from tests.common.helpers import TEST_ROOT
+
+
+MODEL_NAMES = [
+    'resnet18',
+    'mobilenet_v2',
+    'inception_v3',
+    'googlenet',
+    'vgg16',
+    'densenet121'
+]
+
+MODELS = [
+    models.resnet18(),
+    models.mobilenet_v2(),
+    models.inception_v3(),
+    models.googlenet(),
+    models.vgg16(),
+    models.densenet121(),
+]
+
+INPUT_SHAPES = [
+    [1, 3, 224, 224],
+    [1, 3, 224, 224],
+    [1, 3, 224, 224],
+    [1, 3, 224, 224],
+    [1, 3, 224, 224],
+    [1, 3, 224, 224],
+]
+
+INPUT_KEYS = [
+    "input.1",
+    "input.1",
+    "input.1",
+    "input.1",
+    "input.1",
+    "input.1",
+]
+
+TEST_CASES = [
+    pytest.param(name, model, shape, key)
+    for name, model, shape, key in zip(MODEL_NAMES, MODELS, INPUT_SHAPES, INPUT_KEYS)
+]
+
+
+@pytest.fixture
+def load_low_opset_model():
```

Comment:
why do you need @pytest.fixture here? 
Also change the function name. It exports model and then loads it, which is not described in the name

Reply:
Done